substrate/runtime/primitives/src/generic.rs

```diff
@@ -235,6 +235,108 @@ impl<Item> traits::Digest for Digest<Item> where
 }
 
 
+/// Generic digest item that is able to encode/decode 'system' digest items and
```

Comment:
I think this is better described as "opaque" rather than "generic" since any non-explicitly defined entries must be stored in it as a simple vector.

Reply:
Changed description a bit - is it OK now?